apps/condo/domains/ticket/components/BaseTicketForm/useTicketValidations.tsx

```diff
@@ -1,6 +1,12 @@
 import { Rule } from 'rc-field-form/lib/interface'
 import { useIntl } from '@core/next/intl'
 
+const NON_DIGITS_REGEXP = /\D/g
+
+// TODO: @akarjakin - move to apropriate place and add imports
+const MAX_PHONE_LENGTH = 15
+const MIN_PHONE_LENGTH = 11
```

Comment:
May be better to move those constants to `apps/condo/domains/ticket/constants/restrictions.ts`?

Reply:
fixed